app/assets/stylesheets/components/_footer.sass

```diff
@@ -0,0 +1,11 @@
+$Footer-white: Theme-color(background, footer)
```

Comment:
You shouldn't make references to any visual detail. In this case, I'm talking about the `white` part.
Please read [this](http://nicolasgallagher.com/about-html-semantics-front-end-architecture/) ASAP